src/__test__/components/data-processing/data-integration/CalculationConfig.test.jsx

```diff
@@ -118,7 +119,7 @@ describe('Data Integration Calculation Config', () => {
       </Provider>,
     );
 
-    component.find(Select).at(0).getElement().props.onChange('seuratv3');
+    act(() => { component.find(Select).at(0).getElement().props.onChange('seuratv3'); });
```

Comment:
isn't the idea to use the `waitForComponentToPaint ` function in here?

Reply:
Its for when a component updates a field on its initial render, the function is not required here, wrapping a click in act() is enough for this